monkestation/code/modules/nabbers/code/_nabbers.dm

```diff
@@ -0,0 +1,209 @@
+#define NABBER_DAMAGE_ONBURNING 5 //Yeah it needed a nerf.
+#ifdef TRAIT_HARD_SOLES //Checks to see if this trait exists.
+/datum/species/nabber //If Hard_soles is detected, apply it to their inherent_traits. Cross-Testmerge compatability!
+	inherent_traits = list(
```

Comment:
Why, by the way, is this species in the grayscale folder, and not in a separate code folder for species?
This can create confusion and make it more difficult to work with species.